packages/block-library/src/post-content/edit.js

```diff
@@ -45,40 +51,104 @@ function ReadOnlyContent( {
 	);
 }
 
-function EditableContent( { context = {} } ) {
-	const { postType, postId } = context;
+function EmptyContentPlaceholder( { context, onClose, openInserter } ) {
+	const { postType } = context;
+	const label =
+		'page' === postType
+			? __( 'This page’s content is empty' )
+			: __( 'This post’s content is empty' );
```

Comment:
Should we have a fallback string for when the post type is neither a post nor a page? I.e. something like:

```js
let label = __( 'This content is empty' );
if ( 'page' === postType ) {
    label = __( 'This page’s content is empty' );
} else if ( 'post' === postType ) {
    label = __( 'This post’s content is empty' );
}
```